src/decorators/exclude.decorator.ts

```diff
@@ -0,0 +1,23 @@
+import { defaultMetadataStorage } from '../storage';
+import { ExcludeOptions } from '../interfaces';
+
+/**
+ * Marks property as excluded from the process of transformation. By default it excludes the property for both
```

Comment:
I would suggest to take the chance to add to this description that this decorator may also be applied to classes.

Reply:
Fixed.